### ISSUE TITLE
stellarium the free open source planetarium has stopped working

### ISSUE THREAD

**Issue by salodidan** [Stellarium/stellarium#890]
**I have a problem and this message appears

Windows 10

the message : 
(stellarium the free open source planetarium has stopped working)**

![23445](https://user-images.githubusercontent.com/59202508/71415157-65d47500-266b-11ea-9125-0c56a522469f.jpg)

**Comment by gzotti** [comment]
https://github.com/Stellarium/stellarium/wiki/FAQ#reporting-a-crash

**Comment by serastrof** [comment]
It happened to me too, with Stellarium 0.20.1 for Windows64bit.

S.O. windows7ultimate 64bit SP1, recently clean re-installed.
Windws update all gone ok.

Actions taken: uninstalled Stellarium 64bit and installed at 32bit.
The program starts, but hangs on the home screen with the default landscape in full screen. It does not accept commands.
Ctrl + Alt + Del, the task becomes iconic and an error message appears in the background:

msvcr120.dll file not found, reinstall the software.

I searched the net, finding this article:
https://www.groovypost.com/howto/fix-visual-c-plus-plus-redistributable-windows-10/

I followed the instructions. Installed visual C++ redistributable x86 and 64bit too.

Now everything looks OK.
